bip-0077.md

```diff
@@ -0,0 +1,704 @@
+                https://gnusha.org/pi/bitcoindev/3C0A6E4C-444E-4E75-829C-1A21D8EE40E0@ngould.dev/#t
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-08-08
+  License: BSD-2-Clause
+  Requires: 21, 78, 173, 174
+```
+
+## Copyright
+
+This BIP is licensed under the 2-clause BSD license.
+
+## Abstract
+
+Payjoin lets Bitcoin senders and receivers interact to make batched
+transactions.
+
+This document proposes a second, backwards-compatible, asynchronous version of
+the Payjoin protocol described in [BIP 78](bip-0078.mediawiki). An untrusted
+third-party "directory server" replaces the requirement
+for a receiver to host a secure public endpoint for interactions. HTTP clients
+access the directory server using an asynchronous protocol and authenticated,
+encrypted payloads. The design preserves complete Payjoin receiver
+functionality, including payment
+output substitution, is preserved. Authenticated encryption depends only on cryptographic
+primitives available in Bitcoin Core. Requests use [Oblivious
+HTTP](https://www.ietf.org/rfc/rfc9458.html) (OHTTP) to
+prevent the directory and other Payjoin clients from linking requests to client
+IP addresses.
+
+## Motivation
+
+Satoshi Nakamoto pointed out one specific privacy risk in the
+[whitepaper](https://bitcoin.org/en/bitcoin-paper),
+that transactions with multiple inputs "necessarily reveal that
+their inputs were owned by the same owner."
+Payjoin addresses that risk, the _common-input-ownership heuristic_,
+by making it practical to spend inputs owned by multiple parties
+in one transaction.
+
+While addressing Bitcoin's primal privacy risk, Payjoin *input* batching
+also improves on the widespread non-interactive *output* batching practice
+deployed by exchanges. When combined, the same movement of funds can use
+less block weight and save fees.
+
+A natural application of Payjoin would be to combine
+getting paid with consolidating UTXOs into one transaction. But Payjoin
+can also secure [transaction
+cut-through](https://bitcointalk.org/index.php?topic=281848.0),
+allowing a sender to transfer funds to a receiver who also transfers
+funds to a third party in the same transaction. For example, deposits to an
+exchange may "cut through" in a single transaction that also satisfies
+withdrawals instead of with a second transaction that spends the deposited
+funds. Payjoin enables more blockspace-efficient transactions that
+reduce fees while addressing privacy risks.
+
+However, BIP 78's requirements have proven to be an
+obstacle to adoption. Version 1 receivers must host a secured
+public-facing HTTP server. Mobile and web environments limit the ability
+to fulfil such a requirement. Version 1 also requires synchronous
+communication. Both sender and receiver must be online simultaneously.
+Wallet developers [
+regard](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-January/018358.html)
+these requirements as barriers to Payjoin adoption.
```

Comment:
@DanGould I remember discussing that Version 1 and Version 2 are distinct from BIP 78/BIP 77 (I'm forgetting the nuance). In lieu of an explanation, I'm left to conclude that by version 1 we mean BIP 78. It doesn't seem we mention "versions" at all anywhere other than here.